data/assets/scene/digitaluniverse/backgroundradiation.asset

```diff
@@ -33,7 +33,7 @@ local COBE = {
     FadeInThreshold = 0.4
   },
   GUI = {
-    Name = "1990 COBE CMB",
+    Name = "COBE",
```

Comment:
What's the reason for changing your mind?  You suggested to change it from COBE to 1990 COBE CMB in https://github.com/OpenSpace/OpenSpace/commit/babbb4f97c37c68aa26e77b7917180cde2fc2153#diff-5297116b943caeb4f4f994d9be25c49bab9ec51c183c75dff7b8ff94f0c62c48R36

(same for WMAP and Planck)

Reply:
Because I'm going to order them, as mentioned above.